cfn-resources/cluster/mongodb-atlas-cluster.json

```diff
@@ -341,6 +341,10 @@
         "NONE"
       ]
     },
+    "GlobalClusterSelfManagedSharding": {
+      "description": "Set this field to configure the Sharding Management Mode when creating a new Global Cluster. When set to false, the management mode is set to Atlas-Managed Sharding. This mode fully manages the sharding of your Global Cluster and is built to provide a seamless deployment experience. When set to true, the management mode is set to Self-Managed Sharding. This mode leaves the management of shards in your hands and is built to provide an advanced and flexible deployment experience. This setting cannot be changed once the cluster is deployed",
```

Comment:
description is fine but I'm working in adding a little bit more of detail about how to configure, it should be fast, i'll keep you posted.
Please wait for merge, or alternatively you can merge and do the doc changes in another PR

Reply:
@EspenAlbert I have a proposal doc in this PR: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2348
WDYT about waiting until doc team reviews?

Reply:
Sounds good 👍

Reply:
doc is already reviewed in that PR, you can already take the doc from there

Reply:
Nice. Updated in [781c02c](https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/1079/commits/781c02ce2c66d3646210a28804c9791a539a0b37)
Need a restamp